CHANGES

```diff
@@ -2,6 +2,13 @@ Version 8.16 (Unreleased)
 -------------------------
 - Added data migration to create UserEmail objects for users whose primary emails did not have them
 - Time series data (used by graphs and other features) is now updated when groups are merged.
+- Added distributions to the release system to better support mobile apps
+
+Schema Changes
+~~~~~~~~~~~~~~
+
+- Added ``Distribution`` model
+- Added ``Event.dist`` column
```

Comment:
That's what happens when you don't think. It is a column but on `ReleaseFile`. fixed.